README.md

```diff
@@ -8,38 +8,28 @@ Yawoen company has hired you to implement a Data API.
 
 **1 - Load company data in a database**
 
-Read data from CSV file and load into the database to create a catalog named **companies**.
+Read data from CSV file and load into the database to create an entity named **companies**.
 
 This catalog should contain the following fields: id, company name and zip code. 
```

Comment:
"catalog" name is duplicated here, needs change too.

Reply:
thanks!